infra/app/app-config/main.tf

```diff
@@ -6,6 +6,8 @@ locals {
   has_database                    = true
   has_incident_management_service = false
 
+  feature_flags = ["foo", "bar"]
```

Comment:
Just to make sure I understand, to add a new FF, an engineer would add it to this list, then due to the code in other app-config files (specifically infra/modules/feature-flags/access-policy.tf I think?), we automatically pull in flags from here on deploy?